x-pack/plugins/apm/public/components/alerting/fields.tsx

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { EuiSelect, EuiExpression, EuiFieldNumber } from '@elastic/eui';
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiSelectOption } from '@elastic/eui';
+import { EuiFieldNumberProps } from '@elastic/eui';
+import { getEnvironmentLabel } from '../../../common/environment_filter_values';
+import { PopoverExpression } from './ServiceAlertTrigger/PopoverExpression';
+
+export function ServiceField({ value }: { value?: string }) {
```

Comment:
This file contains the commonly-used alerting fields. I intend for these fields to be almost 1:1 in their use across alerts. If they start to diverge we should abandon this approach (but I don't think that'll be the case)